bin/genome-env-next

```diff
@@ -0,0 +1,432 @@
+#!/bin/bash
+
+# if this was found via PATH as opposed to a path to executable
+THIS="$(basename "$0")"
```

Comment:
or, since we're explicitly using bash, ${0##*/} is ~~ the result of s!.*/!! on $0 (greedily eat everything until the last slash)
no need to change it, just sharing since you asked :)